pkg/terraform/terraform.go

```diff
@@ -119,8 +125,13 @@ func unpackAndInit(dir string, platform string) (err error) {
 	defer lpDebug.Close()
 	defer lpError.Close()
 
+	os.Setenv("TF_CLI_CONFIG_FILE", filepath.Join(dir, "terraform.rc"))
+
+	// XXX: This is only here for debugging CI
+	os.Setenv("TF_LOG", "trace")
```

Comment:
We lost setting the `TERRAFORM_LOCK_FILE_PATH` environment variable.

Reply:
Doh!